naps/platform/beta_platform.py

```diff
@@ -22,6 +22,36 @@ def connect_mainboard(self):
             lvds=["68 70", "74 76", "82 84", "92 94", "93 91", "89 87"]
         )
 
+        # these pins are split between the two connectors so we refer to them directly by FPGA site name
+        lvds_p = "H15 K16 J20 G19 F19 G17 J18 K17 L16 K19 M17 M19 L19 B19 F16 D19 V16 V20 H16 V17 R16 Y18 T20 N18 U14 T16 U18 V15 W14 T11 Y16 T14"
```

Comment:
I think it would be cleaner to have this use the connectors, maybe using the `JX{0,1}_0:{num}` syntax, but its fine for now.

Reply:
I updated it to do this with a clearer table.